tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.ServiceTest/IntegrationTests/ConfigUnitTests.cs

```diff
@@ -89,6 +89,52 @@ public void GetAllUIConfigs()
             Assert.True(result.Status > 0);
         }
 
+
+        [Fact]
+        public void GetAllUIConfigsForSPO()

```

Comment:
A unit test case to test whether the SPO UI config.js file will get created in catalog site collection or not?